fluent/src/parser.js

```diff
@@ -121,60 +121,42 @@ class RuntimeParser {
    */
   getMessage() {
     const id = this.getPrivateIdentifier();
-    let attrs = null;
 
     this.skipInlineWS();
 
-    let ch = this._source[this._index];
-
-    let val;
-
-    if (ch === '=') {
+    if (this._source[this._index] === '=') {
       this._index++;
+    }
 
-      this.skipInlineWS();
+    this.skipInlineWS();
 
-      if (this._source[this._index] === '\n') {
-        this.skipBlankLines();
-        if (this._source[this._index] === ' ') {
-          this.skipInlineWS();
-          val = this.getPattern();
-        }
-      } else {
-        // This is a fast-path for the most common
-        // case of `key = Value` where the value
-        // is in the same line as the key.
-        val = this.getPattern();
```

Comment:
This fast path wasn't needed because it's essentially almost the same as what `getPattern` does. Overall, I'm seeing a slight speed-up with my patch, around -7%.